docs/docs/features/slashing.md

```diff
@@ -3,7 +3,7 @@ sidebar_position: 4
 ---
 
```

Comment:
@mpoke It's not clear how valuable this doc is. It seems we could get rid of it altogether and interested readers can look a the corresponding ADRs?

Reply:
Imo it's nice to have a top level doc outside of ADRs that outlines the current slashing functionality, but this doc could easily be outdated